app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -11,18 +15,10 @@ def evaluation_needed
     list_items(AuctionQuery.new.pending_acceptance)
   end
 
-  def complete_and_successful
-    list_items(AuctionQuery.new.complete_and_successful)
-  end
-
   def payment_needed
     list_items(AuctionQuery.new.payment_needed)
   end
 
-  def rejected
```

Comment:
removed rejected from needs attention --hard to tell because the prototype is so different now, but it seems like Michael didn't create a story around complete and successful or rejected auctions so I figure we no longer want them on this page (feels like they never should have been here, anyway)